test/specs/elements/Header/HeaderHeadings-test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import H1 from 'src/elements/Header/HeaderH1'
+import H2 from 'src/elements/Header/HeaderH2'
+import H3 from 'src/elements/Header/HeaderH3'
+import H4 from 'src/elements/Header/HeaderH4'
+import H5 from 'src/elements/Header/HeaderH5'
+import H6 from 'src/elements/Header/HeaderH6'
+
+const headers = {
+  h1: H1,
+  h2: H2,
+  h3: H3,
+  h4: H4,
+  h5: H5,
+  h6: H6,
+}
+
+describe('HeaderHeadings', () => {
+  Object.keys(headers).forEach((key) => {
+    describe(`Header.${key.toUpperCase()}`, () => {
+      const sdClass = `sd-header-${key}`
+      const Component = headers[key]
+      const wrapper = mount(<Component />)
+
+      it(`renders a ${key} element`, () => {
+        wrapper.should.have.tagName(key)
+      })
+
+      it(`adds the ${sdClass} class`, () => {
+        wrapper.should.have.className(sdClass)
+      })
+    })
```

Comment:
All components should conform to the base requirements.  Especially the `H*` components as they are exposed to the user.

When I add:
```js
describe.only('HeaderH1', () => {
  common.isConformant(HeaderH1)
})
```
I get this output:

```shell
  HeaderH1
    ✔ constructor name matches filename "HeaderH1"
    ✔ is exported as a component or sub component
    ✔ spreads user props
    ✔ handles events transparently
    _meta
      ✔ is a static object prop
      library
        ✔ is defined
        ✔ is a META.library
      name
        ✔ is defined
        ✔ matches the filename
      parent
        ✔ matches some component name
      type
        ✔ is defined
        ✔ is a META.type
    className (common)
      ✔ has the Stardust className "sd-header-h1"
      ✔ applies user's className to root component
```